shared/src/androidTest/kotlin/co/touchlab/kampkit/TestUtilAndroid.kt

```diff
@@ -5,8 +5,16 @@ import androidx.test.core.app.ApplicationProvider
 import co.touchlab.kampkit.db.KaMPKitDb
 import com.squareup.sqldelight.android.AndroidSqliteDriver
 import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
 
 internal actual fun testDbConnection(): SqlDriver {
-    val app = ApplicationProvider.getApplicationContext<Application>()
-    return AndroidSqliteDriver(KaMPKitDb.Schema, app, "kampkitdb")
+    // Try to use the android driver (which only works if we're on robolectric).
+    // Fall back to jdbc if that fails.
+    return try {
+        val app = ApplicationProvider.getApplicationContext<Application>()
+        AndroidSqliteDriver(KaMPKitDb.Schema, app, "kampkitdb")
+    } catch (exception: IllegalStateException) {
+        JdbcSqliteDriver(JdbcSqliteDriver.IN_MEMORY)
+            .also { KaMPKitDb.Schema.create(it) }
+    }
```

Comment:
I thought about just moving everything off robolectric, but also figured this might be a useful trick to have in here in case someone else finds it useful.

Reply:
Weird but I like it.